tools/block-generator/generator/make_transactions.go

```diff
@@ -20,15 +20,34 @@ import (
 	"encoding/binary"
 
 	"github.com/algorand/go-algorand/data/basics"
-	"github.com/algorand/go-algorand/data/transactions"
+	txn "github.com/algorand/go-algorand/data/transactions"
+	"github.com/algorand/go-algorand/data/txntest"
 	"github.com/algorand/go-algorand/protocol"
 )
 
-func (g *generator) makeTxnHeader(sender basics.Address, round, intra uint64) transactions.Header {
+// ---- header / boilerplate ----
+
```

Comment:
this diff is confusing. I didn't actually modify `makeTxnHeader` except for renaming the `transactions` import (&rarr; `txn`). But I did introduce `makeTestTxn` which is very similar to `makeTxnHeader` and confused the differ.